pseudomonas_analysis/nbconverted/Viz_E-GEOD-33245_patterns.py

```diff
@@ -23,181 +25,245 @@
 grp_1v2_file = "generic_gene_summary_E-GEOD-33245_1v2.tsv"
 grp_1v3_file = "generic_gene_summary_E-GEOD-33245_1v3.tsv"
 grp_1v4_file = "generic_gene_summary_E-GEOD-33245_1v4.tsv"
+grp_1v5_file = "generic_gene_summary_E-GEOD-33245_1v5.tsv"
 
 # Summary data using raw value of test statistic to get directionality
 grp_1v2_raw_file = "generic_gene_summary_E-GEOD-33245_1v2_raw.tsv"
 grp_1v3_raw_file = "generic_gene_summary_E-GEOD-33245_1v3_raw.tsv"
 grp_1v4_raw_file = "generic_gene_summary_E-GEOD-33245_1v4_raw.tsv"
+grp_1v5_raw_file = "generic_gene_summary_E-GEOD-33245_1v5_raw.tsv"
 
 
-# ## Compare 1v2 and 1v3 grouping
-# Within experiment E-GEOD-33245 we can perform 2 comparisons:
-# * Group 1v2: WT vs crc mutant
-# * Group 1v3: WT vs cbr mutant
+# ## Create dataframe to compare trends
+# We are going to merge data across different conditions. For example, we will merge `grp_1v2` and `grp_1v3` to use for plotting later in this notebook. Deb can look at these tables to find *things of interest* as we start looking into how to use our computational predictions of generic and specific genes. 
 # 
-# We can then compare the test statistic between these two groups above. We hope to see that,
-# * Genes 1v3  > 1v2 are genes that change more in 1v3 than 1v2 and we guess are specific to 1v3 (high z-score)
-# * Genes 1v3 < 1v2 are genes that change more in 1v2 than 1v3 and we guess are specific to 1v2 (high z-score)
-# * Genes on the 1v3 = 1v2 line are those genes that change equally in both and we guess are generic genes (low z-score)
+# She would like a dataframe with 1v2 and 1v3, 1v3 and 1v4.
```

Comment:
Are 1, 2, 3, 4, and 5 here just different datasets, or data collected in different conditions? Maybe you already have this information documented somewhere, I'm just forgetting.

Reply:
Good question. I will definitely add a comment here.

1,2,3,4,5 refer to different conditions/comparisons within the E-GEOD-33245 experiment. So 1 refers to samples comparing WT vs crc mutant.